.github/actions/k3d/action.yaml

```diff
@@ -0,0 +1,94 @@
+name: k3d
```

Comment:
@egegunes should we have all these actions?

Reply:
https://github.com/percona/percona-postgresql-operator/pull/379/commits/bc785d590629a5f3d85ee15b6dd5a07deb003165